### DISCUSSION TITLE
Has anyone tried to add Fleet Telemetry, having used the proxy to obtain Fleet API?

### DISCUSSION THREAD

**Discussion by abreingan** [llamafilm/tesla-http-proxy-addon#96] [Q&A] [answered]
Hi and apologies in advance for the number of questions I seem to be coming up with, and this is somewhat off subject. 
I am currently running a combination of the Fleet API (thanks again llamafilm) and TeslaMate, and becoming concerned that the combination might cause Tesla to cancel the account (use of original API?)
There is what looks like a good description of running a Telemetry server at https://dev.to/patrickdemers6/fleet-telemetry-streaming-data-from-your-tesla-2i97 and I wonder if anyone has gone down this path, and if so can they give any tips on which bits are common to both, and which need to be re-done.
To me it seems likely the creating the developer account and the CSR request are common, and the car already has the Fleet Key installed, so it looks as if I just need to find a way to run the server.
Does that make sense or am I being grossly optimistic?
Thanks in advance
Al

**Comment by llamafilm** [comment, marked as answer]
This addon does not generate a CSR because it’s not needed for the API Proxy. I haven’t tried that app but I guess it should work with the same developer account.
The car can hold multiple keys, so I’d probably suggest making a separate key for each app.
The developer account should work for both.
